host/rootfs/etc/s6-linux-init/run-image/service/getty-ttyLP0/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/execlineb -P
+# SPDX-License-Identifier: EUPL-1.2+
+# SPDX-FileCopyrightText: 2020-2021 Alyssa Ross <hi@alyssa.is>
```

Comment:
Please use `# SPDX-FileCopyrightText: 2022 Unikie` instead for new files and add the same line for existing files with existing copyright when you are contributing new.

Reply:
Ok, got it. It's fixed now.